### DISCUSSION TITLE
Best Practices for Implementing Google Analytics Code in Version 2.x

### DISCUSSION THREAD

**Discussion by kouohhashi** [payloadcms/payload#4926] [Q&A] [answered]
I want to insert the Google Analytics code in version 2.x, what is the most recommended way to do this?

Is it best practice to copy and paste it into the head section?

**Comment by zubricks** [comment, marked as answer]
Hey @kouohhashi you could take a look at our [website repo ](https://github.com/payloadcms/website/blob/main/src/app/layout.tsx)for some pointers. We've implemented GA and it works as expected (if you're using NextJS on the frontend that is).

**Reply by kouohhashi** [reply]
Thanks!

**Reply by Meet57** [reply]
@zubricks 
Please update all the website repo pointers.

**Comment by andrei-aceweb** [comment]
Hi! I'm wondering about the same, and what is the best practice in the latest version of payload? (3.41.0)
Is there any best-practice example for that? The link in the accepted answer is not valid anymore unfortunately
